src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -114,4 +114,8 @@ export class AppConfig extends AbstractAppConfig {
     return this.config.pay_bulk_scan_url;
   }
 
+  public getBannersUrl(): string {
```

Comment:
Use this instead:
https://github.com/hmcts/ccd-case-management-web/blob/2b476547b12f579ac1668474bd752dd78a1fbc32/src/app/app.config.ts#L166

And refrain from using double quotes for string values please.